File: 우현/탐욕/18312_시각.py
n, k = map(int, input().split())

count = 0

for h in range(n+1):
  if h < 10:
    h = '0'+str(h)
  
  for min in range(60):
    if min < 10:
      min = '0'+str(min)
    
    for sec in range(60):
      if sec < 10:
        sec = '0'+str(sec)

      if str(k) in str(h)+str(min)+str(sec):
        count += 1

print(count)

File: 소영/Baekjoon/Baekjoon/구현/18312_시각.swift
//
//  18312_시각.swift
//  Baekjoon
//
//  Created by ohtt on 2022/05/16.
//

func 시각() {
  let read = readLine()!.split(separator: " ").map{ Int($0)! }
  let hour = read[0]
  let number = read[1]
  var count = 0

  for hour in 0...hour {
    for min in 0..<60 {
      for second in 0..<60 {
        if hour % 10 == number || hour / 10 == number ||
            min % 10 == number || min / 10 == number ||
            second % 10 == number || second / 10 == number {
          count += 1
        }
      }
    }
  }

  print(count)
}
